//python -m grpc_tools.protoc -I . --python_out=. --grpc_python_out=. sayhello.proto
syntax = "proto3";


message RequestMsg {
    int32 user_id = 1;
}

message ResponseMsg {
    int32 id = 1;
    string title = 2;
}



service SayHello {
    rpc SayHello (RequestMsg) returns (ResponseMsg);
}
